source/Playnite/App/PlayniteApplication.cs

```diff
@@ -442,15 +440,17 @@ public async void StartUpdateCheckerAsync()
                     {
                         if (updater.IsUpdateAvailable)
                         {
-                            CurrentNative.Dispatcher.Invoke(() =>
+                            var updateTitle = ResourceProvider.GetString("LOCUpdaterWindowTitle");
+                            var updateBody = ResourceProvider.GetString("LOCUpdateIsAvailableNotificationBody");
+                            if (!Current.IsActive)
                             {
-                                var model = new UpdateViewModel(
-                                    updater,
-                                    new UpdateWindowFactory(),
-                                    new ResourceProvider(),
-                                    Dialogs);
-                                model.OpenView();
-                            });
+                                NotifyInWindows(updateTitle, updateBody);
+                            }
+                            else
+                            {
+                                Api.Notifications.Add(new NotificationMessage("UpdateAvailable", updateBody,

```

Comment:
I think we should add internal notification even if application is not active. In case that user dismissed the Windows notification and later opens UI he should still see that update is available.

Reply:
resolved in the recent commits below